src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveReferencesToAliasesPassTest.php

```diff
@@ -101,6 +101,56 @@ public function testResolveFactoryService()
         $this->assertSame('factory', $foo->getFactoryService());
     }
 
+    public function getAliasVisibility()
+    {
+        return array(
+            'private' => array(false),
+            'public' => array(true),
+        );
+    }
+
+    /**
+     * @dataProvider getAliasVisibility
+     *
+     * @param bool $public
+     */
+    public function testResolveNestedDefinitionWithAliasReferences($public)
+    {
+        $container = new ContainerBuilder();
+
+        $container->register('original_dep', '\stdClass');
```

Comment:
is 2nd arg used at all? If not, what about removing it? if yes, leading `\` should be removed (same below)

Reply:
didn't even know we can use it without specifiying a classname